### PR TITLE
Enable code formatter to enforce consistent whitespace style 

### LINKED ISSUE #446: Script formatting should allow setting for space between keyword and opening paren
For instance, I prefer to see:
```powershell
if (Test-Path foo) {
   "Foo exists!"
}
```
instead of
```powershell
if(Test-Path foo) {
   "Foo exists!"
}
```
Same would apply to other keywords: `elseif`, `for`, `foreach`, `while`, `switch`, etc.

### PR DESCRIPTION
**Pull request by kapilmb**
Resolves #446.

### CODE REVIEW

File: src/features/DocumentFormatter.ts [hunk 1]
```diff
@@ -139,6 +139,7 @@ class PSDocumentFormattingEditProvider implements DocumentFormattingEditProvider
     private readonly ruleOrder: string[] = [
         "PSPlaceCloseBrace",
         "PSPlaceOpenBrace",
+        "PSUseWhitespace",
```
Comment (daviwil) [inline src/features/DocumentFormatter.ts]: Should this be named PSUseConsistentWhitespace?

Comment (kapilmb) [inline src/features/DocumentFormatter.ts]: I was in a bit of a limbo on whether to have "consistent" in its name. But looks like "PSUseConsistentWhitespace" sounds more consistent with the "PSUseConsistentIndentation" rule. On the other hand all the rules enforce some or the other kind of consistency and so having the "consistent" word seemed verbose (And I was even thinking of changing "PSUseConsistentIndentation" to "PSUseIndentation", but that would be a breaking change for PSSA). At this point, I personally don't have any preference but if you think "PSUseConsistentWhitespace" sounds better, I can change it to that.

Comment (daviwil) [inline src/features/DocumentFormatter.ts]: I think consistency would be nice between the two names.  "PSUseIndentation/Whitespace" doesn't sound specific enough to me; the "Consistent" designation helps with that.  I've tried thinking of other names and I can't come up with anything better at the moment.

Comment (kapilmb) [inline src/features/DocumentFormatter.ts]: Fixed!

Also made corresponding changes to PowerShell/PSScriptAnalyzer#702

Comment (daviwil) [inline src/features/DocumentFormatter.ts]: Thanks man!

**Comment by kapilmb** [comment]
@daviwil Thanks!

**Pull request merged**